### PR TITLE
READY : List saved games 

### PR DESCRIPTION
**Pull request by olgZZZ**

### CODE REVIEW

**Comment by dmvict** [comment]
Please, write some tests. And test the browser version also.

File: module/core/src/lib.rs [hunk 1]
```diff
@@ -278,8 +278,8 @@ impl Board
     {
       if sq % 8 == 0
       {
-        s.push( char::from_digit( rank, 10 ).unwrap());
-        s.push_str( " | " );
+        s.push(  char::from_digit(  rank, 10  ).unwrap() );
```
Comment (dmvict) [inline module/core/src/lib.rs:281]: Do not add extra spaces, please.

File: module/core/src/lib.rs [hunk 2]
```diff
@@ -834,3 +834,27 @@ pub fn board_der< 'de, D : Deserializer< 'de > >( d : D ) -> Result< Board, D::E
...
+pub fn list_saved_games() -> Option< Vec< std::path::PathBuf > >
+{
+  let mut game_list = vec![];
+  let saves_dir = fs::read_dir( SAVES_FOLDER_NAME );
+
+  if let Ok( paths ) = saves_dir
+  {
+    for path in paths
+    {
+      game_list.push( path.unwrap().path() );
```
Comment (dmvict) [inline module/core/src/lib.rs:851]: I think that routine should print filenames, not paths. 

Improve it or explain, please.

File: module/core/tests/core_tests.rs [hunk 3]
```diff
@@ -86,4 +86,17 @@ fn test_make_random_move()
   assert_eq!( game.status(), GameStatus::Continuing );
   let turn_after = game.current_turn();
   assert_ne!( turn_befor, turn_after );
-}
\ No newline at end of file
+}
+
+#[ test ]
+fn test_list_saved_games()
```
Comment (dmvict) [inline module/core/tests/core_tests.rs:92]: Please, add test with non empty list.
